web/pages/index.tsx

```diff
@@ -37,30 +39,38 @@ import PostCreationModal from "../components/PostCreationModal/PostCreationModal
 import Feed from "../components/Feed/Feed";
 
 function Home() {
-  const { loading, authorized } = useAuth();
+  let { loading, authorized } = useAuth();
+
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Instead of creating new state in `index`, refactor the `loading` state in `context/auth`. ~~This is an antipattern, but~~ export `setLoading` so that it is available via `useAuth()`. This will allow you to override the `loading` state regardless of what's happening in `context/auth`. Our existing context makes user, loading, authorized, and userData available, so add `setLoading` to those (see below for what it looks like now):
```jsx
<AuthContext.Provider value={{ user, loading, authorized, userData }}>
      {children}
</AuthContext.Provider>
```